File: primerTrimestre/introduccion_programacion_python/3repetitivas_py/ej3rep.py
# Programa: ej3rep.py
# Propósito: Algoritmo que pida caracteres e imprima ‘VOCAL’ si son vocales y ‘NO VOCAL’ en caso contrario,
# el programa termina cuando se introduce un espacio.
#
# Autor: Antonio Quesada
# Fecha: 30/10/2019.
#
# Variables a usar:
#   * caracter
#
# Algoritmo:
# Pedimos dato al usuario
# Mientras el caracter sea igualo distinto a a,e,i,o,u es una vocal.
# Si no es una consonante.

# Petición de datos.
caracter = str(input("Introduce un carácter: "))

# Ciclo while y salida.
while caracter!=" ":
    if caracter=="a" or caracter=="e" or caracter=="i" or caracter=="o" or caracter=="u" or caracter=="A" or caracter=="E" or caracter=="I" or caracter=="O" or caracter=="U":
        print ("Es una vocal") 
        break
    else:
        print ("No es una vocal")
        break
    

File: primerTrimestre/java basico/3repetitivas/Ej3rep.java
/**Algoritmo que pida caracteres e imprima ‘VOCAL’ si son vocales y ‘NO VOCAL’ en caso contrario, el programa termina
 * cuando se introduce un espacio.
 *
 * @author Antonio Quesada
 */

import java.util.Scanner;

public class Ej3rep {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    String caracter;
    
    //Pedimos caracteres.
    System.out.println("Introduce un caracter: ");
    caracter = s.nextLine();
    
    //Mientras el caracter sea igualo distinto a a,e,i,o,u es una vocal.
    while (caracter!=" ") {
      if (caracter.equals("a") || caracter.equals("e") || caracter.equals("i") || caracter.equals("o") || caracter.equals("u") || caracter.equals("A") || caracter.equals("E") || caracter.equals("I") || caracter.equals("O") || caracter.equals("U")) {
        System.out.println("Es una vocal");
        break; //Si es una vocal rompe el ciclo y termina.
        
      } else {
          System.out.println("No es una vocal");
          break; //Si no es una vocal rompe el ciclo y termina.
         
        }
      }
    }

  }


